core/app.go

```diff
@@ -343,6 +349,9 @@ func (a *App) handlePolling() {
 		}
 		a.Telemetry.Serve()
 	}
+	if a.ControlSocket != nil {
```

Comment:
We should start the control socket before we start any other work rather than here at the end of `handlePolling` Otherwise we won't be listening to the `preStart` handler or (if particularly unlucky) any of the other pollables if they manage to get scheduled and run before we can start listening on the socket.

If you look a bit higher-up in `Run` you'll see `handleSignals`. We should start the listener there, as that's what this control plane is going to replace. (Let's start it immediately afterwards for now; it'll be gone when we're done here.)